### PR TITLE
Add conformance-ppc64le dashboard

### PR DESCRIPTION
**Pull request by mkumatag**

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -6690,7 +6690,7 @@ dashboards:
   dashboard_tab:
   - name: conformance
     test_group_name: ppc64le-conformance
-    description: conformance test results for ppc64le
+    description: conformance test results for ppc64le ****This is moved under conformance dashboard, WILL BE REMOVED FROM HERE IN THE FUTURE****
```
Comment (dims) [inline testgrid/config.yaml]: any reason not to just remove it?

Comment (mkumatag) [reply testgrid/config.yaml]: Many people are referring this link, I don't want to surprise them immediately.. will remove after sometime.

File: testgrid/config.yaml [hunk 2]
```diff
@@ -7951,6 +7951,13 @@ dashboards:
     test_group_name: arm64-conformance-vexxhost-cluster
     description: Runs conformance tests by using kubetest against latest version of kubernetes on VexxHost"
 
+# ppc64le dashboard
+- name: conformance-ppc64le
+  dashboard_tab:
+  - name: Periodic ppc64le conformance test on local cluster
+    test_group_name: ppc64le-conformance
+    description: Runs conformance tests against latest version of kubernetes on local ppc64le cluster"
```
Comment (BenTheElder) [inline testgrid/config.yaml]: What is local?

File: testgrid/config.yaml (hunk 1 above)
Comment (BenTheElder) [reply testgrid/config.yaml]: You can leave it in both 🤷‍♂️

File: testgrid/config.yaml (hunk 2 above)
Comment (mkumatag) [reply testgrid/config.yaml]: kubeadm cluster running in the IBM lab

**Pull request merged**